src/inventory.cc

```diff
@@ -4273,7 +4254,7 @@ int inventoryOpenLooting(Object* looter, Object* target)
             break;
         }
 
-        if (keyCode == KEY_UPPERCASE_A) {
+        if (keyCode == KEY_UPPERCASE_A || keyCode == KEY_LOWERCASE_A) {
```

Comment:
This lets you use `a` to `Take All`.  It seems perhaps already intended, but making it uppercase means you'd need to use shift-a in practice.